corehq/toggles/__init__.py

```diff
@@ -1030,14 +1030,23 @@ def _ensure_valid_randomness(randomness):
     namespaces=[NAMESPACE_DOMAIN]
 )
 
+DYNAMICALLY_UPDATE_SEARCH_RESULTS = StaticToggle(
+    'dynamically_update_search_results',
+    "In case search with split screen case search enabled, search results update when a search field is updated"
+    " without requiring the user to manually press a button to search.",
+    TAG_CUSTOM,
+    help_link='https://confluence.dimagi.com/display/USH/Split+Screen+Case+Search',
+    namespaces=[NAMESPACE_DOMAIN],
+)
+
 SPLIT_SCREEN_CASE_SEARCH = StaticToggle(
     'split_screen_case_search',
     "Split screen case search: In case search, show the search filters in a sidebar on the left and the results"
     " on the right.",
     TAG_CUSTOM,
     help_link='https://confluence.dimagi.com/display/USH/Split+Screen+Case+Search',
     namespaces=[NAMESPACE_DOMAIN],
-    parent_toggles=[SYNC_SEARCH_CASE_CLAIM]
+    parent_toggles=[SYNC_SEARCH_CASE_CLAIM, DYNAMICALLY_UPDATE_SEARCH_RESULTS]
```

Comment:
should this not be inverted? I think `DYNAMICALLY_UPDATE_SEARCH_RESULTS` is a 'child' of `SPLIT_SCREEN_CASE_SEARCH`

Reply:
This is what I originally had put, but for the functionality I wanted through testing that if `SPLIT_SCREEN_CASE_SEARCH` was enabled I wanted `DYNAMICALLY_UPDATE_SEARCH_RESULTS` to be enabled.-- my understanding is that this is the relationship that I would need for that-- is that a correct assumption?

Reply:
Sure, that is correct - but it means that you could enable `DYNAMICALLY_UPDATE_SEARCH_RESULTS` without having `SPLIT_SCREEN_CASE_SEARCH` enabled.

If you always want `DYNAMICALLY_UPDATE_SEARCH_RESULTS` enabled with `SPLIT_SCREEN_CASE_SEARCH` it seems like maybe you should just use the same toggle?